<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectManagement v.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>von (KW)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_from_kw">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>53</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_from_year">
        <property name="minimum">
         <number>2017</number>
        </property>
        <property name="maximum">
         <number>2027</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>bis (KW)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_to_kw">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>53</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_to_year">
        <property name="minimum">
         <number>2017</number>
        </property>
        <property name="maximum">
         <number>2027</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_new_sector">
        <property name="text">
         <string>laden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_source">
        <property name="text">
         <string>Quelle</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_reload">
        <property name="text">
         <string>aktualisieren</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_edit">
        <property name="text">
         <string>bearbeiten</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainContainer">
      <item>
       <layout class="QHBoxLayout" name="barLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pb_new">
          <property name="font">
           <font>
            <pointsize>21</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
